src/classBased/components/TodoContainer.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable semi */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/sort-comp */
+/* eslint-disable react/state-in-constructor */
```

Comment:
[OPTIONAL]
- Avoid deactivating too many linters, this may lead to a weak code structure. Don't worry about it now, but keep it in mind for the next projects.